include/nbl/builtin/hlsl/complex.hlsl

```diff
@@ -24,7 +26,7 @@ struct complex_t
 };
 
 template<typename T>
-complex_t<T> add(const complex_t<T> a, const complex_t<T> b)
+complex_t<T> plus(const complex_t<T> a, const complex_t<T> b)
 {
     complex_t<T> result;
 
```

Comment:
we already have struct functors for those in `nbl/builtin/hlsl/functional` you need to add an explicit specialization

Reply:
Marking this as resolved as this is now an operator